packages/it/index.js

```diff
@@ -1,97 +1,99 @@
-var compose = require('@stamp/compose');
-var Shortcut = require('@stamp/shortcut');
-var isStamp = require('@stamp/is/stamp');
-var isString = require('@stamp/is/string');
-var isObject = require('@stamp/is/object');
-var isFunction = require('@stamp/is/function');
-var merge = require('@stamp/core/merge');
-var assign = require('@stamp/core/assign');
-
-var concat = Array.prototype.concat;
-function extractFunctions() {
-  var fns = concat.apply([], arguments).filter(isFunction);
+/* eslint-disable no-use-before-define */
+
+'use strict';
```

Comment:
AFAIK `use strict` is not needed in node 8 and later.

Reply:
Now that you mention it, `use strict` is not necessary for node scripts and commonjs modules. I'd need to check with Typescript still.

I have to habit of always writing and running in strict mode